### ISSUE TITLE
Integrate Real Chat

### ISSUE THREAD

**Issue by imanolGOMI** [sudharti/socify#31]
Something like this. http://josephndungu.com/tutorials/gmail-like-chat-application-in-ruby-on-rails

@sudharti  when you merge the changes master to develop and then create new master with all features i do a fork and try code this issue. ;)

**Comment by scaffeinate** [comment]
@w0www Really cool feature bro 👍

**Comment by Rtech2014** [comment]
Hii sudharti if you permit me i can do messenger like chat in socify

**Comment by scaffeinate** [comment]
@Rtech2014 Yeah feel free to send me a PR. I ll be glad to review and merge that in

**Comment by scaffeinate** [comment]
@Rtech2014 If you are gonna follow that tutorial link above then I must say they have used private pub in that. I was thinking the new Rails 5 realtime feature would be great to try on.

**Comment by bdmorales** [comment]
Any update on the chat feature?

**Comment by mishraak** [comment]
Is somebody already working or willing to work on this? I would like to work on this feature, but I do not want duplication of efforts.

**Comment by imanolGOMI** [comment]
I think that nothing is working in this feature.

**Comment by imanolGOMI** [comment]
@scaffeinate the proyect is dead or alive?

**Comment by scaffeinate** [comment]
@mishraak You are more than welcome to contribute. Just send me a PR.

**Comment by scaffeinate** [comment]
@w0www Sorry bro, Got no time. You can always contribute.

**Comment by imanolGOMI** [comment]
@scaffeinate dont worry mate! ;) this project is awesome maybe the community can contribute to improvement some features ;) i dont have time now but have this project as favorite to in a future do something

**Comment by scaffeinate** [comment]
@w0www Thanks dude! I will try my best to maintain this project 👍
